packages/devtools_app_shared/lib/src/service/service_utils.dart

```diff
@@ -48,17 +48,21 @@ extension VmServiceExtension on VmService {
     await forEachIsolateHelper(this, callback);
   }
 
-  /// Posts an event to jump to code at the specified [line] and [column] in the
-  /// specified [fileUri]. The [source] should indicate the tool and/or feature
-  /// that is making the jump to code request.
+  /// Posts an event to jump to code.
+  ///
+  /// The event directs an IDE to move to the [line] and [column] in the
+  /// specified [fileUriString], which is expected to be a proper file URI (i.e.
+  /// starts with "file://"). The [source] should indicate the tool and/or
+  /// feature (i.e. in the format "someTool.someFeature") that is making the
+  /// jump to code request.
```

Comment:
add one more section. If there are no IDEs listening for this event, then this will be a no-op.